int16 joint1
int16 joint2
int16 joint3
int16 joint4
int16 joint5
